src/components/SettingsModal.tsx

```diff
@@ -102,7 +107,6 @@ const SettingsModal = ({ isOpen, onClose }) => {
           <Switch
             defaultChecked={colorMode === 'dark'}
             size="lg"
-            value={colorMode === 'dark'}
```

Comment:
Because `value` is useless. That [props controls when value this HTML Form element will submit](https://chakra-ui.com/docs/components/form/switch#props). I only noticed because its type is `string | number`

Reply:
Did you check that it actually didn't affect functionality? I thought that was needed to make the inputs controlled

Reply:
Maybe chakra does something special for that though, in which case yeah let's get rid of it

Reply:
Yeah, I did. Nothing changed. Everything works just like before.